packages/autorest.typescript/test/unit/restLevelClient/generateReadmeFile.spec.ts

```diff
@@ -23,6 +23,7 @@ describe("generateReadmeFile", () => {
         scopeName: "azure-rest",
         version: "1.0.0-beta.1"
       },
+      flavor: "azure",
```

Comment:
just wondering why we need this setting, i believe the package scope name would guide to azure sdk?

Reply:
That doesn't work here since the logic relies on `host.getValue` which doesn't work in this context. So we need to be explicit.